### DISCUSSION TITLE
error 404

### DISCUSSION THREAD

**Discussion by Xaylain** [TeKrop/overfast-api#188] [Q&A]
Hello, I have tried to run your project and when I do a search by battle tag it gives me this error

{
  "error": "Player not found"
}

**Comment by TeKrop** [comment]
Hello! 

Can you give more details, including your request? For example, this URL is working : https://overfast-api.tekrop.fr/players?name=TeKrop

And this one as well, even if there is no result : https://overfast-api.tekrop.fr/players?name=Aaaabbbn

Make sure your profile is public so it can be retrieved by the API. You can use this page to search for your profile page, to make sure your profile is available : https://overwatch.blizzard.com/en-us/search/?q=Test

**Reply by Xaylain** [reply]
I have searched for my id called ''September 15'' and it found it with the battle tag, but when I search in player_id it gives me this error
{
  "error": "Player not found"
}
and my profile in the game is open to the public and it is not in streamer mode

**Reply by TeKrop** [reply]
Hmm that's weird indeed, your profile is appearing on search results, but when clicking on the profile there is a 404 error on Blizzard page, which is the source for the API... I can't do anything for this, and I don't know why it's happening 😕

**Reply by Xaylain** [reply]
I would like to know something else, does the API support special characters in battle tags?

**Reply by TeKrop** [reply]
Yes it does, I already loaded profiles with special characters 👍

**Reply by Xaylain** [reply]
It seems that it works with special characters but when I search for player_id it doesn't find the same error, the account is from Europe, does that have anything to do with it?

**Reply by Xaylain** [reply]
this is the battle tag （４：２０）#2959

**Reply by TeKrop** [reply]
Hmm maybe, but there isn't any filtering made on the player region in the API, and on Blizzard pages as well as far as I know 🤔

**Comment by idawud91** [comment]
I'm running into the same issue, and I'm pretty sure it's Blizzard, not the API. Starting yesterday, all I could get was the "player not found error". I tried a few others and only about 1 out of every 5 worked. When I tried the Blizzard user profile search, same thing.
